pages/_section/photos/_gallery.vue

```diff
@@ -141,9 +143,27 @@ export default {
       }
     }
   },
+  beforeMount () {
+    const details = navigator.userAgent
+    const regexp = /android|iphone|kindle|ipad/i
+    const isMobileDevice = regexp.test(details)
+
+    const IMG_SCALE_POINT = 1147
+    const IMG_SCALE_MAX = 1098
+
+    const PADDING = isMobileDevice ? 32 : 49
+    const WINDOW_WIDTH = window.innerWidth
+
+    this.imgWidth =
+      WINDOW_WIDTH < IMG_SCALE_POINT ? WINDOW_WIDTH - PADDING : IMG_SCALE_MAX
+  },
```

Comment:
i don't understand why we're trying to do device detection here? Is there a reason to not just use the viewport width?
Also if we're using imgWidth in the template we should add it to the component data object and maybe give it a more descriptive name.

Reply:
yeah... nice... simplified this.